packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -877,6 +884,15 @@ export class ReadAlongComponent {
         } else {
           this.images[i] = null;
         }
+        //load the ids for the end of the page
+        if ("paragraphs" in page) {
+          const paragraphs = (page as Page).paragraphs;
+          const sentence =
+            paragraphs[paragraphs.length - 1].querySelector("s:last-of-type");
+          const word = sentence.querySelector("w:last-of-type");
+          this.endOfPageTags[word.id] = parseFloat(word.getAttribute("dur"));
```

Comment:
This is really weird, why is the duration of the last word becoming the ID for the end of page tag?

Reply:
I am creating a list of the last word spoken on each page, using the ID as the key and the duration of the word to help calculate when to pause the audio.